classes/class-lessons.php

```diff
@@ -42,37 +42,45 @@ public function get_items() {
 	 * @return array
 	 */
 	public function get_remote_api_items() {
-		$cached = \progress_planner()->get_cache()->get( 'lessons' );
-		if ( is_array( $cached ) && ! empty( $cached ) ) {
+
+		/**
+		 * Filter the endpoint url for the lessons.
+		 *
+		 * @param string $endpoint The endpoint url.
+		 */
+		$url = apply_filters(
+			'progress_planner_lessons_endpoint',
+			'https://progressplanner.com/wp-json/progress-planner-saas/v1/free-lessons'
+		);
+
+		$cache_key = 'lessons-' . $url;
```

Comment:
It would probably be safer if we `md5()` the URL...

```suggestion
		$cache_key = 'lessons-' . md5( $url );
```

Reply:
Sure, I left it just because of the debugging - any hash will do the job.